src/python/espressomd/bond_breakage.py

```diff
@@ -55,3 +55,19 @@ def __getitem__(self, key):
 
     def __setitem__(self, key, value):
         return super().__setitem__(self._get_key(key), value)
+
+
+@script_interface_register
+class BondBreakage(ScriptInterfaceHelper):
+    """Bond breakage interface.
+
+    This class provides methods to manage and manipulate bond breakage.
+
+    Methods
+    -------
+    execute(parameters)
+        Executes the bond breakage with the given parameters.
+    """
+
+    _so_name = "BondBreakage::BondBreakage"
+    _so_bind_methods = ("execute",)
```

Comment:
@RudolfWeeber I'm not a big fan of having this feature as a stand-alone class. It currently only works on the main `System` instance, and ignores all other instances. How about making this feature a member method of the `BreakageSpecs` class? Then one could do `system1.bond_breakage.execute()` to execute all specs.

Reply:
moved it.